airbyte-integrations/connectors/source-zendesk-support/source_zendesk_support/manifest.yaml

```diff
@@ -353,6 +349,32 @@ definitions:
       cursor_filter: "start_time"
       primary_key: "id"
 
+  posts_stream:
+    $ref: "#/definitions/base_incremental_stream"
+    name: posts
+    primary_key: "id"
+    incremental_sync:
+      $ref: "#/definitions/cursor_incremental_sync"
+      start_time_option:
+        $ref: "#/definitions/cursor_incremental_sync/start_time_option"
+        field_name: "start_time"
+    retriever:
+      $ref: "#/definitions/retriever"
+      ignore_stream_slicer_parameters_on_paginated_requests: true
+      requester:
+        $ref: "#/definitions/retriever/requester"
+        path: "community/posts"
+      paginator:
+        $ref: "#/definitions/links_next_paginator"
+        pagination_strategy:
+          $ref: "#/definitions/links_next_paginator/pagination_strategy"
+          stop_condition: "{{ config.get('ignore_pagination', False) or not response['meta']['has_more'] }}"
+      record_selector:
+        $ref: "#/definitions/retriever/record_selector"
+        schema_normalization: None
```

Comment:
This is required because schema is a bit loose: for example, instead of integer as `id`, it is `number` which cast fields as float